### PR TITLE
fix #97821: PageSettings remember window resize

### PR DESCRIPTION
**Pull request by salilkhandelwal**
PageSettings will remember the last user resize even after closing

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
See also #2500, we need to make sure these kind of things are done in a consistent manner.

**Comment by salilkhandelwal** [comment]
Are you referring to the spaces I have left?

File: mscore/pagesettings.cpp [hunk 1]
```diff
@@ -81,9 +88,46 @@ PageSettings::PageSettings(QWidget* parent)
 
 PageSettings::~PageSettings()
       {
+      QSettings settings;
+
```
Comment (salilkhandelwal) [inline mscore/pagesettings.cpp]: @Jojo-Schmitz  I think this is the problem you are talking about,  "spaces" (which is disturbing the coding style and consistency).

Comment (WASamK) [inline mscore/pagesettings.cpp]: I think he is referring to having these "remember window resize" functionality in a similar manner in all the windows. I did it in a similar manner to start center(https://github.com/musescore/MuseScore/blob/master/mscore/startcenter.cpp) but my code still have some improvements to be done.(See the comments in #2500)

Comment (Jojo-Schmitz) [inline mscore/pagesettings.cpp]: @salilkhandelwal :no, @WASamK: yes ;-)

Comment (Jojo-Schmitz) [inline mscore/pagesettings.cpp]: @salilkhandelwal  but yes, there's some newlines too much here too

**Comment by salilkhandelwal** [comment]
I will remove them

**Comment by Jojo-Schmitz** [comment]
A rebase is needed

**Comment by lasconic** [comment]
See #2752

**Pull request closed without merging**